### PR TITLE
AI personality, Mario tutorial: update to reflect current SW

### PR DESCRIPTION
**Pull request by c0ffeeca7**
## Proposed change

AI personality, Mario tutorial: update to reflect current software
- Google AI and OpenAI integration pages: add info that only exposed entities can be controlled
- Add related topics

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [x] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logos and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

## Summary by CodeRabbit

- **New Features**
  - Updated the voice assistant personality creation tutorial to support both OpenAI and Google Generative AI integrations.
  - Added a new related link for creating an assistant personality with AI.

- **Documentation**
  - Clarified instructions and descriptions for controlling Home Assistant and generating images with OpenAI.
  - Updated link text to reflect broader AI integration.

### CODE REVIEW

**Comment by c0ffeeca7** [comment]
@coderabbitai full review

**Comment by c0ffeeca7** [comment]
> @coderabbitai full review

@coderabbitai full review

File: source/_integrations/google_generative_ai_conversation.markdown [hunk 1]
```diff
@@ -86,9 +86,9 @@ Safety settings:
 Allows you to ask Gemini Pro or Gemini Pro Vision to generate content from a prompt consisting of text and optionally images.
 This service populates [response data](/docs/scripts/service-calls#use-templates-to-handle-response-data) with the generated content.
 
-| Service data attribute | Optional | Description                                    | Example             |
-| ---------------------- | -------- | ---------------------------------------------- | ------------------- |
-| `prompt`               | no       | The prompt for generating the content.         | Describe this image |
+| Service data attribute | Optional | Description                                     | Example             |
+| ---------------------- | -------- | ----------------------------------------------- | ------------------- |
+| `prompt`               | no       | The prompt for generating the content.          | Describe this image |
```
Comment (c0ffeeca7) [inline source/_integrations/google_generative_ai_conversation.markdown]:
```suggestion
| Service data attribute | Optional | Description                                    | Example             |
| ---------------------- | -------- | ---------------------------------------------- | ------------------- |
| `prompt`               | no       | The prompt for generating the content.         | Describe this image |
```

File: source/_integrations/google_generative_ai_conversation.markdown [hunk 2]
```diff
@@ -42,7 +42,7 @@ Instructions:
   description: Instructions for the AI on how it should respond to your requests. It is written using [Home Assistant Templating](/docs/configuration/templating/).
 
 Control Home Assistant:
-  description: If the model is allowed to interact with Home Assistant
+  description: If the model is allowed to interact with Home Assistant.
```
Comment (c0ffeeca7) [inline source/_integrations/google_generative_ai_conversation.markdown]:
```suggestion
  description: If the model is allowed to interact with Home Assistant
```

File: source/_includes/asides/voice_navigation.html [hunk 3]
```diff
@@ -22,7 +22,7 @@ <h1 class="title delta">{% active_link /voice_control/ Voice assistants %}</h1>
       <li>{% active_link /voice_control/custom_sentences/ Custom sentences %}</li>
       <li>{% active_link /voice_control/aliases/ Using aliases %}</li>
       <li>{% active_link /voice_control/using_tts_in_automation/ Using Piper TTS in automations %}</li>
-      <li>{% active_link /voice_control/assist_create_open_ai_personality/ Creating a personality with OpenAI %}</li>
+      <li>{% active_link /voice_control/assist_create_open_ai_personality/ Creating a personality with AI %}</li>
```
Comment (c0ffeeca7) [inline source/_includes/asides/voice_navigation.html]:
```suggestion
      <li>{% active_link /voice_control/assist_create_open_ai_personality/ Creating a personality with OpenAI %}</li>
```

File: source/voice_control/worlds-most-private-voice-assistant.markdown [hunk 4]
```diff
@@ -5,6 +5,8 @@ related:
     title: Creating a Cloud assistant
   - docs: /voice_control/voice_remote_local_assistant/
     title: Creating a local assistant
+  - docs: /voice_control/assist_create_open_ai_personality/
+    title: Creating an assistant personality with AI
```
Comment (c0ffeeca7) [inline source/voice_control/worlds-most-private-voice-assistant.markdown]:
```suggestion
```

File: source/voice_control/worlds-most-private-voice-assistant.markdown [hunk 5]
```diff
@@ -122,7 +124,7 @@ The [debug information](/voice_control/troubleshooting#view-debug-information) s
 
 ## Other troubleshooting steps
 
-Are things still not working as expected? 
+Are things still not working as expected?
```
Comment (c0ffeeca7) [inline source/voice_control/worlds-most-private-voice-assistant.markdown]:
```suggestion
Are things still not working as expected? 
```

File: source/voice_control/assist_create_open_ai_personality.markdown [hunk 6]
```diff
@@ -1,18 +1,37 @@
...
+    title: Google Generative AI integration
+  - docs: /integrations/openai_conversation/
+    title: OpenAI Conversation integration
+  - docs: /voice_control/voice_remote_expose_devices/
+    title: Exposing entities to Assist
+    
 ---
 
-You can give your voice assistant personality by using OpenAI. This requires an OpenAI account. For what we do in this tutorial, the free trial option is sufficient. No need to leave your credit card information.
+You can give your voice assistant personality by using an AI conversation agent. Currently, this works with the **OpenAI Conversation** or the **Google Generative AI** integration. At time of writing (June 7, 2024), Google was 14x cheaper than OpenAI, but OpenAI is better at answering non-smart home questions.
```
Comment (c0ffeeca7) [inline source/voice_control/assist_create_open_ai_personality.markdown]:
```suggestion
You can give your voice assistant personality by using an AI conversation agent. Currently, this works with the **OpenAI Conversation** or the **Google Generative AI** integration.
```

**Review by frenck — APPROVED** [review]
Thanks, @c0ffeeca7 👍 

../Frenck

**Pull request merged**